src/views/text_view.rs

```diff
@@ -318,31 +352,29 @@ impl TextView {
     fn compute_rows(&mut self, size: Vec2) {
         let size = if self.wrap { size } else { Vec2::max_value() };
 
-        let mut content = self.content.lock().unwrap();
+        let mut content = self.content.content.lock().unwrap();
         if content.is_cache_valid(size) {
             return;
         }
 
         // Completely bust the cache
         // Just in case we fail, we don't want to leave a bad cache.
         content.size_cache = None;
+        content
+            .content_cache
+            .replace(content.content_value.borrow().clone());
```

Comment:
Let's use `Arc::clone(...)` rather than `.clone()` to be clear about what we're cloning here.